views/index.jade

```diff
@@ -14,7 +15,7 @@ block content
           input#email.form-control(type='email', placeholder='Enter Email', name='email', required='')
         .form-group
           label.control-label(for='git_url') GITURL:
-          input#git_url.form-control(type='text', placeholder='Enter URL of Github Repository', name='git_url')
+          select#git_url.form-control(type='text', placeholder='Enter URL of Github Repository', name='git_url')
```

Comment:
I queried **if editable select could be used** here, we would not need to use `typeahead`. It would reduce the codebase. If we can use editable select with the same functionality as `typeahead` without introducing more code, wouldn't it be better?